core/domain/topic_services.py

```diff
@@ -802,15 +802,11 @@ def _are_nodes_valid_for_publishing(
             story_nodes: list(dict(str, *)). The list of story nodes dicts.
 
         Raises:
-            Exception. The story node doesn't contain any exploration id or the
-                exploration id is invalid or isn't published yet.
+            Exception. The exploration id is invalid or isn't published yet.
```

Comment:
... or corresponds to an exploration which isn't published yet.

Reply:
Done.